src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int income;
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder report = new StringBuilder();
```
при неймінгу змінних краще керуватись її ціллю а не типом